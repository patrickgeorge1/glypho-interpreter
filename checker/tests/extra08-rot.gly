YYY^...0rrr'44{4S000x+++nzzz