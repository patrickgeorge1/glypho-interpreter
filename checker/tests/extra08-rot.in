1
-456364456
-98653457643456982343243973857542689346543216523164523
