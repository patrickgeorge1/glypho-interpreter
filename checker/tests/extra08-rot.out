-456364456
1
-98653457643456982343243973857542689346543216523164523
